errors/Readme.md

```diff
@@ -195,7 +195,7 @@ Note: Studio has two workflows:
 Electron app: Credentials from the UI → Introspection → Prisma schema → Valid Prisma project
 Web app: `prisma2 dev` → Provides Prisma schema i.e a Valid Prisma project
 
-Since studio uses Photon for query building. It relays the same error messages as Photon. Additionally, it relays the following errors from the SDK: `P3000`.
+Since studio uses Prisma Client JS for query building. It relays the same error messages as Prisma Client JS. Additionally, it relays the following errors from the SDK: `P3000`.
```

Comment:
```suggestion
Since studio uses Prisma Client JS for query building it relays the same error messages as Prisma Client JS. Additionally, it relays the following errors from the SDK: `P3000`.
```